style.css

```
@@ -0,0 +1,483 @@
+    border-radius: 4px;
+}
+
+#main-block::before {
+    content: "";
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 50%;
+    background-image: url("./images/215911.gif");
+    background-repeat: no-repeat;
+    background-size: 100%;
+    opacity: 0.1;
+    z-index: -1;
+}
+
+#main-block::after {
+    background-color: #1E1E28;
+    content: "";
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 50%;
+    background: linear-gradient(to bottom, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.1));
+    pointer-events: none;
+    z-index: 0;
+}
+
+.main-message {
+    background-image: url("./images/215911.gif");
+    background-repeat: no-repeat;
+    background-size: 100%;
+    background-position: center;
+    border-radius: 5px;
+    font-family: "Poppins";
+    position: relative;
+    z-index: 1;
+    height: 300px;
+    margin-top: 50px;
+    margin-bottom: 25px;
+    margin-left: 30px;
+    margin-right: 30px;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    box-shadow: 0px 0px 5px 1px rgba(0, 0, 0, 0.1);
+}
+
+
+.main-message::after {
+    border-radius: 4.3px;
+    content: "";
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: linear-gradient(to bottom, rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5));
+    pointer-events: none;
+    z-index: 0;
+}
+
+.main-message > h1 {
+    color: white;
+    font-size: 45px;
+    font-weight: bold;
+    text-align: center;
+    z-index: inherit;
+}
+
+.main-message > h2 {
+    color: white;
+    font-size: 20px;
+    font-weight: 600;
+    margin-top: 50px;
+    text-align: center;
+    z-index: inherit;
+}
+
+.main-message > h2 > img {
+    width: 30px;
+}
+
+.main-message > h3 {
+    border-right: #fff solid 2px;
+    color: white;
+    font-size: 15px;
+    margin-bottom: 20px;
+    overflow: hidden;
+    text-align: center;
+    width: max-content;
+    white-space: nowrap;
+    z-index: inherit;
+}
+
+.button-mode {
+    display: flex;
+    margin: 15px auto;
+}
+
+.button-mode > button {
+    background-color: white;
+    border: none;
+    box-shadow: 0px 0px 5px 1px rgba(0, 0, 0, 0.5);
+    border-radius: 5px;
+    color: black;
+    font-family: "Poppins";
+    padding: 5px;
+    margin: 0 20px;
+    z-index: 3;
+}
+
+.typing-animation {
+    animation: blinkCursor 500ms steps(40) infinite normal, typing 4s steps(40) 1s normal both;
+}
+
+@keyframes blinkCursor {
+    from {
+        border-right-color: #fff;
+    }
+    to {
+        border-right-color: transparent;
+    }
+}
+
+@keyframes typing {
+    from {
+        width: 0;
+    }
+    to {
+        width: 20.5em;
+    }
+}
+
+#skills-tools-block > h2 {
+    color: white;
+    font-size: 15px;
+    font-weight: 400;
+}
+
+.theme-btn {
+    height: 30px;
+    width: 100px;
+}
+
+.theme-btn > i {
+    margin-bottom: 10px;
+}
+
+.theme-btn > p {
+    color: white;
+    font-size: 10px;
+}
+
+.lenguage-btn {
```

Acho que voce quis dizer "language", nao?

```suggestion
.language-btn {
```

---

KKKK, ajustado, thankssssssss